### ISSUE TITLE
git submodule update --init fails

### ISSUE THREAD

**Issue by coder543** [vadimcn/cargo-pgo#1]
For some reason, `git submodule update --init` fails.

    fatal: reference is not a tree: c8a8767c56ad3d3f4eb45c87b95026936fb9aa35
    Unable to checkout 'c8a8767c56ad3d3f4eb45c87b95026936fb9aa35' in submodule path 'profiler-rt/compiler-rt'

I guess something got deleted from the repo being pulled from?

**Comment by vadimcn** [comment]
Yeah, a lot has changed since last time I touched this.  For example, rust toolchain now includes its own profiler runtime (albeit guarded by a feature flag), so building profiler-rt might not be necessary.
I didn't have the time lately to explore new developments.
